functions-and-scope.js

```diff
@@ -16,6 +17,23 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+//Stappenplan:
+//Stap 1 loopen door array
+//Stap 2 length toevoegen aan loop zodat hij werkt ongeacht de hoeveelheid cijfers
+//stap 3 Aantal cijfer 8 of hoger selecteren
+//Stap 4 Cijfers opslaan
+//stap 4 cijfers loggen
+
+const cumLaudeList = [];
```

Comment:
Let hier op dat je nu een const declareert, terwijl we juist willen dat dit een overschrijfbaar variabel is (let).